pallets/streaming/src/lib.rs

```diff
@@ -76,13 +81,16 @@ pub struct Stream<T: Config> {
     start_time: Timestamp,
     // Stop Time
     stop_time: Timestamp,
+    // The current status of the stream
+    status: StreamStatus,
 }
 
 #[frame_support::pallet]
 pub mod pallet {
     use super::*;
 
     use weights::WeightInfo;
+    pub const MAX_STREAM_LEN: usize = 1024;
```

Comment:
probably too big, I would suggest 32 to start

Reply:
okay